content/en/platform/corda/5.0-dev-preview-2/getting-started/fast-feedback-with-the-simulator/fast-feedback-with-the-simulator.md

```diff
@@ -23,9 +23,11 @@ As Simulator is not executing Corda code, the error messages are likely to be di
 
 ## Using Simulator
 
-Simulator is designed to be used within a normal Java or Kotlin testing framework. In this documentation, we describe using JUnit. [MyFirstFlow](../first-flow.html) has a corresponding test class `MyFirstFlowTest` that demonstrates how to use Simulator. This file is in `/src/test/kotlin/com.r3.developers.csdetemplate.MyFirstFlowTest.kt` in the [CSDE](../cordapp-standard-development-environment/csde.html) template repository.
+Simulator is designed to be used within a normal Java or Kotlin testing framework. In this documentation, we describe using JUnit. [MyFirstFlow](../first-flow.html) has a corresponding test class `MyFirstFlowTest` that demonstrates how to use Simulator. This file is in `/src/test/kotlin/com.r3.developers.csdetemplate.MyFirstFlowTest.kt` or `/src/test/java/com.r3.developers.csdetemplate.MyFirstFlowTest.java` in the [CSDE](../cordapp-standard-development-environment/csde.html) template repository.
```

Comment:
```suggestion
Simulator is designed to be used within a normal Java or Kotlin testing framework. In this documentation, you can learn how to use JUnit. [MyFirstFlow](../first-flow.html) has a corresponding test class `MyFirstFlowTest` that demonstrates how to use Simulator. This file is in `/src/test/kotlin/com.r3.developers.csdetemplate.MyFirstFlowTest.kt` or `/src/test/java/com.r3.developers.csdetemplate.MyFirstFlowTest.java` in the [CSDE](../cordapp-standard-development-environment/csde.html) template repository.
```

Reply:
Changed to "This documentation describes using JUnit"